examples/features/deadline/client/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary client is an example client.
+// Binary client demonstrates how to set deadlines for RPCs and handle
```

Comment:
Nit: 

// Binary client demonstrates how to set deadlines for RPCs and how to handle
// deadline-exceeded errors.

Reply:
Done